modules/programs/topgrade.nix

```
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.topgrade;
+
+  tomlFormat = pkgs.formats.toml { };
+
+in {
+
+  meta.maintainers = [ maintainers.msfjarvis ];
```

```suggestion
  meta.maintainers = [ hm.maintainers.msfjarvis ];
```

`lib.maintainers` is for maintainers in NixOS' nixpkgs and `lib.hm.maintainers` for HM's maintainers file.

You can add yourself to nixpkgs' maintainers list without maintaining a package IIRC. You only need to open a PR 😃

---

All fixed, thanks for your review!